src/services/dataService.js

```diff
@@ -602,13 +602,17 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
 //            pollInterval: in milliseconds, how long to wait between polling the server
 //                    only applies if poll=true.  Default is 5000.
 //            http:   similar to .get, etc. at this point, only used to pass http.params for filtering
+//            suppressDigest: will skip the $apply & avoid triggering a digest loop
```

Comment:
`skipDigest` maybe? Or possibly `skipApply` to be consistent with `$timeout` naming. (No strong preference.)